src/panels/lovelace/cards/hui-thermostat-card.ts

```diff
@@ -93,13 +93,18 @@ export class HuiThermostatCard extends hassLocalizeLitMixin(LitElement)
           <div id="tooltip">
             <div class="title">${computeStateName(stateObj)}</div>
             <div class="current-temperature">
-              <span class="current-temperature-text">${
-                stateObj.attributes.current_temperature
-              }
-                <span class="uom">${
-                  this.hass.config.unit_system.temperature
-                }</span>
+            ${ 
+              stateObj.attributes.current_temperature 
+              ? html`
+              <span class="current-temperature-text">
+                ${stateObj.attributes.current_temperature}
+                <span class="uom">
+                  ${this.hass.config.unit_system.temperature}
+                </span>
               </span>
+              `
+              : html`<span class="current-temperature-text"></span>`
```

Comment:
We don't need to render anything?

Reply:
Why render the span if nothing is to go there?

Reply:
Because otherwise the layout ist messed up.